Blogplace.Tests.Integration/Tests/ArticlesTests.cs

```diff
@@ -203,6 +223,25 @@ public async Task Delete_ShouldNotBeDeletedByOtherUser()
         article.Should().NotBeNull();
     }
 
+    [Test]
+    public async Task Delete_ShouldNotBeDeletedWithoutPermission()
```

Comment:
`WithoutPermission` bardziej `AsAnotherUser`, z tego co widze po kodzie to próbujesz usuwać jako inny użytkownik